ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/acm-group-du-sno-ranGen.yaml

```diff
@@ -34,24 +34,43 @@ policies:
     - path: source-crs/ClusterLogForwarder.yaml
       patches:
       - spec:
+          serviceAccount:
+            name: logcollector
           outputs:
-            - type: "kafka"
-              name: kafka-open
-              # below url is an example
-              url: tcp://10.46.55.190:9092/test
+            - name: default-lokistack
+              type: lokiStack
+              lokiStack:
+                target:
+                  name: logging-loki
+                  namespace: openshift-logging
+                authorization:
+                  token:
+                    from: serviceAccount
+              tls:
+                ca:
+                  key: service-ca.crt
+                  configMapName: openshift-service-ca.crt
```

Comment:
The configuration should be set up for forwarding only. No local storage/observability for logs. Same for the copies below...

Reply:
Loki and COO dropped across the board